app/src/pages/outside/loginPage/serviceVersionsBlock/serviceVersionsBlockWithData.jsx

```diff
@@ -36,22 +36,38 @@ export class ServiceVersionsBlockWithData extends Component {
   calculateServices = (serviceVersions, latestServiceVersions) => {
```

Comment:
Please use `compositeInfoSelector` in the `connect` decorator to extract necessary app services info